frontend/src/components/GenericModal.vue

```diff
@@ -232,39 +229,24 @@ onUnmounted(() => {
 }
 
 @media (--md) {
-  .modal-header {
-    margin-bottom: 0;
-  }
-
   .modal {
     width: 50rem; /* 800px */
```

Comment:
Can we align that value with the current breaking point? Otherwise we have an intermediate state, where the sides are cut from the modal:

<img width="50%" alt="Image" src="https://github.com/user-attachments/assets/0de452bc-b001-4e40-85c8-bb104250e8fc" />


```suggestion
    width: 48rem; /* 768px */
```